sdk/python/foundation-models/system/import/import_model_into_registry.ipynb

```diff
@@ -402,10 +405,17 @@
    "outputs": [],
    "source": [
     "pipeline_object = model_import_pipeline(\n",
-    "    model_id=MODEL_ID, compute=COMPUTE, task_name=TASK_NAME\n",
```

Comment:
Should this be reverted to make use of MODEL_ID? `MMDETECTION_MODEL_ID` is defined in below sections

Reply:
Fixed in latest